From 9c17f9817a8c38dd4e821749cfbd52d53cbff3ee Mon Sep 17 00:00:00 2001
From: Devajith Valaparambil Sreeramaswamy
 <devajith.valaparambil.sreeramaswamy@cern.ch>
Date: Thu, 31 Oct 2024 09:32:20 +0100
Subject: [PATCH 43/52] Revert "In cases of nested parsing at execution time,
 sometimes we need to do LookupFile without current file lexer."

This reverts commit 15362159093405a115c2d2935dce5630101cc1e9.
---
 interpreter/llvm-project/clang/lib/Lex/PPDirectives.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/interpreter/llvm-project/clang/lib/Lex/PPDirectives.cpp b/interpreter/llvm-project/clang/lib/Lex/PPDirectives.cpp
index 331b9adfa7..a980f4bcba 100644
--- a/interpreter/llvm-project/clang/lib/Lex/PPDirectives.cpp
+++ b/interpreter/llvm-project/clang/lib/Lex/PPDirectives.cpp
@@ -965,7 +965,7 @@ OptionalFileEntryRef Preprocessor::LookupFile(
   // stack, record the parent #includes.
   SmallVector<std::pair<OptionalFileEntryRef, DirectoryEntryRef>, 16> Includers;
   bool BuildSystemModule = false;
-  if (!FromDir && !FromFile && getCurrentFileLexer()) {
+  if (!FromDir && !FromFile) {
     FileID FID = getCurrentFileLexer()->getFileID();
     OptionalFileEntryRef FileEnt = SourceMgr.getFileEntryRefForID(FID);
 
-- 
2.43.5

